scripts/notify/index.js

```diff
@@ -1,6 +1,6 @@
 import { readFile } from 'node:fs/promises';
 import { fileURLToPath } from 'node:url';
-import { globby as glob } from 'globby';
+import glob from 'tiny-glob';
```

Comment:
I would suggest using https://www.npmjs.com/package/tinyglobby, which is even smaller

Reply:
As said in [this comment](https://github.com/withastro/adapters/pull/395#pullrequestreview-2309960412) let's switch altogether in a follow-up :)